content/attacks/posts/2023-11-23-HecoBridge.md

```diff
@@ -47,19 +47,12 @@ HTX:
    - $2,195,836 in ARIX
    - $4,254,541 in KOK
 
-Tron:
-   - $500,000 in TRX
-   - $10.3 million in USDT, 
-   - $2.2 million in USDC, 
-   - $521.7k in RockDAO
-   - approximately $1 million in BTT tokens.
-
 ## Timeline
 
-   - **November 22, 2023, 10:06 UTC:** Initial [funds transfer] (https://etherscan.io/address/0xfc146d1caf6ba1d1ce6dcb5b35dcbf895f50b0c4) begins.
-   - **November 22, 2023, 10:56 UTC:** Heco Bridge begins the process of moving recovery funds into [wallet.](https://etherscan.io/txs)
-   - **November 22, 2023, 12:34 UTC:** Justin Sun announces the attack via X and states all withdrawals and deposits have been temporarily [suspended](https://twitter.com/justinsuntron/status/1727304656622326180) and HTX will compensate for the losses from its hot wallet.
-   - **December 4, 2023:** [Postmortem Analysis](https://olympixai.medium.com/heco-bridge-hack-analysis-64cffda76684) is posted on Medium.
+   - **November 22, 2023 at 10:06 Coordinated Universal Time:** Initial [funds transfer](https://etherscan.io/block/18626577) begins.
```

Comment:
Please recheck the usual formatting for the Timeline events in other articles, for example Stake.com, and change it accordingly. The link is still unrelevant, it shows block, not transaction

Reply:
I'm not sure what you mean by the transaction. Is this where I would put the hash? I can delete the bullet if not.

Reply:
Transaction link corrected. 

Once you or Ev advise me how to move forward with the timeline format, I'll get that fixed. At that point, all the requested corrections will be done and I'll request a review.

Reply:
@evgenydmitriev can you confirm which time format we want to use here? 24hr or 12hr AM/PM?  I had been using 24 hr clocks.

Reply:
It doesn't really matter on the analysis side. LLMs are capable of understanding both.

Reply:
Thank you for the info.